### ISSUE TITLE
test scripts leave a pile of files around

### ISSUE THREAD

**Issue by flihp** [01org/tpm2.0-tools#125]
After running the test_all.sh script the following untracked files are left in my source tree:
```
	actcredential.out
	ak_name_pub.out
	ak_pub.out
	akparse.out
	ek_pub.out
	grep.txt
	hash.in
	hash_out_e_0x000B
	hash_tk_e_0x000B
	mkcredential.out
	nv.test_w
	output_ak
	pcrs
	random.out
	secret.data
	src/.tpm2_createprimary.c.swp
	test/.test_tpm2_takeownership_all.sh.swp
	test/actcredential.out
	test/ak_name_pub.out
	test/ak_pub.out
	test/akparse.out
	test/ek_pub.out
	test/grep.txt
	test/hash.in
	test/hash_out_e_0x000B
	test/hash_tk_e_0x000B
	test/mkcredential.out
	test/nv.test_w
	test/output_ak
	test/pcrs
	test/random.out
	test/secret.data
```

Expecting users / developers to clean these up manually (or to ignore them) isn't reasonable.

**Comment by williamcroberts** [comment]
Yeah this bugged me as well, for now, so they don't accidentally end up in a commit, I propose ignoring them while we fix up the test scripts to not be so messy.

**Comment by williamcroberts** [comment]
We could seperate the tests into two directories:

unit - A place for all the C unit tests
system - A place for all the shell scripts

The wrapper functions for the shell scripts could then just delete anything that ins't a script at the end of the test execution.

**Comment by williamcroberts** [comment]
Possible fixes are on:
https://github.com/01org/tpm2.0-tools/pull/129
https://github.com/01org/tpm2.0-tools/pull/130

We can pick one or the other, or both. On both, 129 needs to move to test/unit

**Comment by williamcroberts** [comment]
Fixed in: f569e46a
